src/main/proto/spine/options.proto

```diff
@@ -1003,3 +1006,23 @@ message CompareByOption {
     // the lower, enums — from the last number value to the 0th value, etc.
     bool descending = 2;
 }
+
+// Defines the error message used if a `set_once` field is set again.
+//
+// Applies only to the fields marked as `set_once`.
+//
+// Example: Using the `(set_once)` option.
+//
+//     message User {
+//         UserId id = 1 [(set_once) = true,
+//                        (if_set_again).error_msg = "The student ID can't be re-assigned."];
+//     }
+//
+message IfSetAgainOption {
+
+    // The default error message for the field.
+    option (default_message) = "The field value can't be re-assigned.";
```

Comment:
Why don't we name the field in the message? We should. We also should tell the current value, and the new one. This saves time in debugging.